core/templates/dev/head/pages/error/Error.js

```diff
@@ -17,12 +17,15 @@
  */
 
 oppia.controller('Error', [
-  '$scope', 'UrlInterpolationService',
+  '$scope', 'PageTitleService', 'UrlInterpolationService',
   function(
-      $scope, UrlInterpolationService) {
+      $scope, PageTitleService, UrlInterpolationService) {
     $scope.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
       '/general/oops_mint.png');
 
     $scope.statusCode = GLOBALS.status_code;
+
+    var title = 'Error ' + $scope.statusCode + ' - Oppia';
+    PageTitleService.setPageTitle(title);
```

Comment:
You don't need the variable for anything just write `PageTitleService.setPageTitle('Error ' + $scope.statusCode + ' - Oppia');`

Reply:
Okay, will do that.